src/Orleans/AssemblyLoader/AssemblyLoader.cs

```diff
@@ -291,15 +81,15 @@ private static Assembly MatchWithLoadedAssembly(AssemblyName searchFor, IEnumera
                 var candidateFullName = assembly.FullName;
                 if (String.Equals(candidateFullName, searchForFullName, StringComparison.OrdinalIgnoreCase))
                 {
-                    return assembly;                    
+                    return assembly;
                 }
             }
             return null;
         }
 
         private static Assembly MatchWithLoadedAssembly(AssemblyName searchFor, AppDomain appDomain)
         {
-            return 
+            return
                 MatchWithLoadedAssembly(searchFor, appDomain.GetAssemblies()) ??
                 MatchWithLoadedAssembly(searchFor, appDomain.ReflectionOnlyGetAssemblies());
```

Comment:
is this function still being used ? I think we are dropping using AppDomain to scan assemblies. 
Just a little confused.